tensorflow/core/ops/ops.pbtxt

```diff
@@ -10313,6 +10313,154 @@ op {
   summary: "Computes gradients of the maxpooling function."
 }
 op {
+  name: "MaxPoolGradGrad"
```

Comment:
Yeah, that worked. I assume you meant something like this:
```c++
REGISTER_OP("MaxPool3DGrad")
    .Input("orig_input: Tinput")
    .Input("orig_output: Tinput")
    .Input("grad: T")
    .Output("output: T")
    .Attr("T: numbertype = DT_FLOAT")
    .Attr("Tinput: numbertype = DT_FLOAT");
```
Compatibility test passes now but I assume that original definition contained mistake. Do we want to preserve backward compatibility for this op?